pkg/apis/runtime/request_stream.go

```diff
@@ -34,7 +34,10 @@ func (r RequestUnidiStream) Write(p []byte) (n int, err error) {
 
 // SendMsg sends the given data to client.
 func (r RequestUnidiStream) SendMsg(data []byte) error {
+	data = append(data, "\n\n"...)
```

Comment:
Why we need two line breaks here?

Reply:
This is used for separating each message, then could decode the event object from stream.